arrocke/gloss-translation
# feat: highlight current verse references in BDB sidebar

## What has changed
- Importing BDB preserves `<a>` tags in lexicon entries.
- Seed is updated with modified BDB.
- Created `bdb-book-ref-names.ts`, which contains the names for books used by the BDB `<a>` tags.
- In the sidebar lexicon tab (for BDB only), references to the current verse are highlighted with the tailwind class `bg-yellow-300`
- At the current moment, verse references in lexicon entries are not clickable.



## Connected Issues

closes #330 
## QA Steps
1. Run `nx run db:prisma migrate reset`
2. Open the web app and select a word (hebrew).
3. Confirm that all references to the current verse are highlighted
4. Change the verse and perform steps 2 and 3 for the newly selected verse


## Post-Deployment


- [x] The seed has already been updated
